examples/basic-oauth/update-metadata.php

```diff
@@ -1,23 +1,32 @@
 <?php
-    
-    if (isset($_REQUEST['update']) && $_REQUEST['update']) {
-        
-        $newMetadata = json_decode($_REQUEST["metadata"], true);
-        
-        $auth0Oauth->updateUserMetadata($newMetadata);
-        
-        echo "<div>UPDATED!!!</div>";
-    }  
-  
-$userInfo = $auth0Oauth->getUser();
-  
+
+use Auth0\SDK\API\Management;
+
+  if (isset($_REQUEST['update']) && $_REQUEST['update']) {
+
+    $managementApi = new Management($auth0->getIdToken(), $domain);
```

Comment:
not yet. Not sure if @chenkie has the update of this quickstart in the backlog. Once we release the docs ans everything related to p2, we should update it. Before that it will bring confusion.